packages/sanity/src/desk/components/pane/PaneHeader.tsx

```diff
@@ -50,9 +52,31 @@ export const PaneHeader = forwardRef(function PaneHeader(
 
   const showTabsOrSubActions = Boolean(!collapsed && (tabs || subActions))
 
+  /*   Used for conditionally rendering border on bottom of the header  */
+  const handleScroll = useCallback((event: Event) => {
+    const target = event.target as HTMLElement
+
+    setIsScrollable(target.scrollHeight > target.clientHeight)
+    setHasScrolledFromTop(target.scrollTop > 0)
+  }, [])
+
+  useEffect(() => {
+    scrollableElement?.addEventListener('scroll', handleScroll)
+    return () => {
+      scrollableElement?.removeEventListener('scroll', handleScroll)
+    }
+  }, [handleScroll, scrollableElement])
+
   return (
     <LayerProvider zOffset={100}>
-      <Root data-collapsed={collapsed ? '' : undefined} data-testid="pane-header" ref={ref}>
+      <Root
+        $borderBottom={isScrollable && hasScrolledFromTop}
+        data-collapsed={collapsed ? '' : undefined}
+        data-testid="pane-header"
+        $hasScrolledFromTop={hasScrolledFromTop}
+        $isContentScrollable={isScrollable}
```

Comment:
See above, no need to pass 3 props to our styled component when the single `$borderBottom` is sufficient!